ambari-server/src/main/java/org/apache/ambari/server/controller/internal/UpgradeResourceProvider.java

```diff
@@ -134,22 +134,27 @@
   public static final String UPGRADE_REQUEST_STATUS = "Upgrade/request_status";
   public static final String UPGRADE_SUSPENDED = "Upgrade/suspended";
   public static final String UPGRADE_ABORT_REASON = "Upgrade/abort_reason";
+  @Deprecated
   public static final String UPGRADE_SKIP_PREREQUISITE_CHECKS = "Upgrade/skip_prerequisite_checks";
+  @Deprecated
   public static final String UPGRADE_FAIL_ON_CHECK_WARNINGS = "Upgrade/fail_on_check_warnings";
 
   /**
    * Skip slave/client component failures if the tasks are skippable.
    */
+  @Deprecated
   public static final String UPGRADE_SKIP_FAILURES = "Upgrade/skip_failures";
```

Comment:
I'd just remove them and not bother deprecating.

Reply:
I already had enough "issues" getting things to work - I didn't want to create more change yet to get this in.  I have another jira where I'll be removing items; I'll include this.